src/features/components/fields/HourInputField.js

```diff
@@ -0,0 +1,105 @@
+function generateHours() {
+  const options = []
+  for (let i = 8; i <= 19; i++) {
+    options.push({
+      text: {
+        type: 'plain_text',
+        text: `${i}:00`,
+        emoji: true,
+      },
+      value: `${i}:00`,
+    })
+    options.push({
+      text: {
+        type: 'plain_text',
+        text: `${i}:15`,
+        emoji: true,
+      },
+      value: `${i}:15`,
+    })
+    options.push({
+      text: {
+        type: 'plain_text',
+        text: `${i}:30`,
+        emoji: true,
+      },
+      value: `${i}:30`,
+    })
+    options.push({
+      text: {
+        type: 'plain_text',
+        text: `${i}:45`,
+        emoji: true,
+      },
+      value: `${i}:45`,
+    })
+  }
+  return options
+}
+
```

Comment:
You are returning 3 times the same object just to modify 2/3 strings. This should be sent from the outside when it's called to make this file easier to use.